pkgs/applications/networking/cluster/k3s/builder.nix

```
@@ -184,6 +185,16 @@ let
     src = k3sRepo;
     vendorHash = k3sVendorHash;
 
+    patches =
+      # Disable: Add runtime checking of golang version
+      lib.optional (lib.versionAtLeast k3sVersion "1.29")
```

```suggestion
      lib.optional (lib.versionAtLeast k3sVersion "1.29")
```
As the other versions also need this, it will be necessary to make a single PR.